tests/platform_tests/args/advanced_reboot_args.py

```diff
@@ -143,3 +143,15 @@ def add_advanced_reboot_args(parser):
             "which helps to keep total runtime within desired limits. Avg time per case: " +\
             "sad(3h45m), multi_sad(5h), sad_bgp(1h5m), sad_lag_member(1h15m), sad_lag(1h15m), sad_vlan_port(1h10m), sad_inboot(1h20m)",
     )
+
+    parser.addoption(
+        "--single_sad_only",
+        action="store_true",
+        default=False
+    )
+
+    parser.addoption(
+        "--multi_sad_only",
+        action="store_true",
+        default=False
```

Comment:
Should set multi_sad_only default value to True?

Reply:
I left it as it is to avoid unexpected skips for others. Vendor needs decide himself and skip one of the suites explicitly 
 using any proposed flag, if he wants to, passing the corresponding flag at test run